pkg/vpcmodel/detailedConn.go

```diff
@@ -109,14 +111,30 @@ func (d *detailedConn) nonTCPAndResponsiveTCPComponent() *connection.Set {
 	return d.tcpRspEnable.Union(d.nonTCP)
 }
 
-// todo: will it still be needed once transformation is completed?
 func (d *detailedConn) string() string {
 	if !d.tcpRspDisable.IsEmpty() {
 		return d.allConn.String() + asterisk
 	}
 	return d.allConn.String()
 }
 
+func (d *detailedConn) detailString() string {
```

Comment:
I would consider renaming this function and adding some documentation, about the changes from the `string()` method.
some comment to indicate that the responsiveness indication is per cube, not per the entire connection string.

Reply:
done